lib/util.js

```diff
@@ -872,11 +879,13 @@ exports.getProxyFromEnv = function (isHttps = true) {
   if (proxyFromEnv.indexOf('://') === -1) {
     Logger.getInstance().info('Util.getProxyEnv: the protocol was missing from the environment proxy. Use the HTTP protocol.');
     proxyFromEnv = 'http' + '://' + proxyFromEnv;
+    isHttps = false;
   }
   proxyFromEnv = new URL(proxyFromEnv);
+  const port = getDefaultPortIfNotSet(proxyFromEnv, isHttps);
   const proxy = {
     host: this.validateEmptyString(proxyFromEnv.hostname),
-    port: Number(this.validateEmptyString(proxyFromEnv.port)),
+    port: Number(this.validateEmptyString(port)),
```

Comment:
I think this validate is no longer needed

Reply:
Ok, fixed